apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -897,6 +905,16 @@ export class InstallManager {
           // Example: "C:\MyRepo\common\temp\npm-local\node_modules\.bin\npm"
           const packageManagerFilename: string = this._rushConfiguration.packageManagerToolFilename;
 
+          const packageManagerEnv: NodeJS.ProcessEnv = process.env;
```

Comment:
Is this the existing behavior?

Reply:
Yes, we the process is launched with the "environment" in which rush was launched.